src/main/java/io/anserini/rerank/lib/CondensedListRelevanceReranker.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Anserini: A toolkit for reproducible information retrieval research built on Lucene
```

Comment:
There was a change to the tagline/header per #680, could you help edit them to be consistent here and in `Rm3Reranker.java`? 

Maybe also change wildcard imports (I'm guessing this is an IDE thing) to individual imports for clarity?